docs/usage/configure-renovate.md

```diff
@@ -63,3 +63,8 @@ Renovate will update your PR description each time it finds changes.
 ## Merge
 
 Once you're done checking and configuring in your Configure Renovate PR, it's time to merge it to enable the real Pull Requests to begin.
+
+## Re-do configuration from scratch
```

Comment:
The new section is added to this page: https://docs.renovatebot.com/configure-renovate/